docs/source/user/installing.rst

```diff
@@ -165,9 +165,15 @@ Source archives of the latest release can also be found on
 `PyPI <https://pypi.org/project/numba/>`_.  In addition to ``llvmlite``, you will also need:
 
 * A C compiler compatible with your Python installation.  If you are using
-  Anaconda, you can install the Linux compiler conda packages ``gcc_linux-64``
-  and ``gxx_linux-64``, or macOS packages ``clang_osx-64`` and
-  ``clangxx_osx-64``.
+  Anaconda, you can use the follwing conda packages:
+
+  * Linux ``x86``: ``gcc_linux-32`` and ``gxx_linux-32``
+  * Linux ``x86_64``: ``gcc_linux-64`` and ``gxx_linux-64``
+  * Linux ``POWER``: ``gcc_linux-ppc64le``
```

Comment:
The C++ compiler `gxx_linux-ppc64le` is also needed.

Reply:
Yup